File: sentimental-mario-less/mario.py
# TODO
from cs50 import get_int


def main():

    while True:
        height = get_int("height: ")
        if height > 0 and height <= 8:
            break

    for i in range(height):
        print(" " * (height - 1 - i), end="")
        print("#"*(i+1))


main()

File: mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int n;
    do
    {

        n = get_int("height:");

    }
    while (n < 1 || n > 8);
//row
    for (int i = 1; i <= n; i++)
    {

//spaces
        for (int k = 0; k < n - i; k++)
        {

            printf(" ");
        }
        for (int j = 0; j < i; j++)
        {
            printf("#");
        }

        printf("\n");
    }
}